tests/test_suites/ShortSystemTests/test_many_parallel_reads.sh

```diff
@@ -0,0 +1,66 @@
+timeout_set 2 minutes
+
+CHUNKSERVERS=12 \
+    USE_RAMDISK=YES \
+    MOUNT_EXTRA_CONFIG="readaheadmaxwindowsize=256,maxreadaheadrequests=30,readworkers=100" \
+    MASTER_CUSTOM_GOALS="8 ec_8_4: \$ec(8,4)"
+    setup_local_empty_saunafs info
+
+cd ${info[mount0]}
+
+# For LongSystemTests
```

Comment:
Consider removing the commented code which is not used in the test.

Reply:
Applied.